### DISCUSSION TITLE
Goals images

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#2026] [General]
Hello

The images of the objectives are not loaded

Why this situation?

![image](https://github.com/open-sdg/open-sdg/assets/117297044/d2a6aa26-1ace-45cc-8ace-4946d642c472)

Thank you

**Comment by christoph-LSN** [comment]
@SandraEliz can you share your site and data repo?

**Reply by SandraEliz** [reply]
Hello

https://github.com/prep-odsmexico/site
https://github.com/prep-odsmexico/data

**Comment by christoph-LSN** [comment]
@SandraEliz and where is the staging site?

**Reply by SandraEliz** [reply]
Hello

The images are already visible
![image](https://github.com/open-sdg/open-sdg/assets/117297044/2a2f4b82-4f94-4323-a048-396a413349ca)


Thank you__

**Comment by SandraEliz** [comment]
Hello

Again the images of the goals are not displayed
Thank you

**Comment by SandraEliz** [comment]
Hello

Since the images are not loaded, I added in the folder assets/img/goals/es/ and assets/img/goals/en/ my own images.

Also change in site/_data/site_config.yml the address where the images come from goal_image_base: '/site/assets/img/goals'

It works fine, but when the changes are passed to production, it marks errors and does not allow the changes to be generated.
![image](https://github.com/open-sdg/open-sdg/assets/117297044/b343dde5-7c34-41ef-b0c2-afb571f66cc9)


Why is this happening?

Thank you

**Reply by brockfanning** [reply]
@SandraEliz That error looks unrelated to goal images. To troubleshoot, I would recommend that you first look at the specific code changes that caused this error to start happening. That should hopefully be a clue about what's going on.
